src/Agent.Worker/TestResults/NunitResultReader.cs

```diff
@@ -267,10 +243,94 @@ private List<TestCaseResultData> FindTestCaseNodes(XmlNode startNode, string hos
             return results;
         }
 
-        public bool AddResultsFileToRunLevelAttachments
+        public string InnerTestSuiteNodeName { get; set; }
+        public string TestCaseNodeName { get; set; }
+        public string RootNodeName { get; set; }
+
+    }
+
+
+    public interface INUnitResultsXmlReader
+    {
+        TestRunData GetTestRunData(string filePath, XmlDocument doc, XmlNode testResultsNode, TestRunContext runContext, bool addResultsAsAttachments);
+    }
+
+    internal class NUnit2ResultsXmlReader: NUnitResultsXmlReader, INUnitResultsXmlReader
+    {
+        public new TestRunData GetTestRunData(string filePath, XmlDocument doc, XmlNode testResultsNode, TestRunContext runContext, bool addResultsAsAttachments)
+        {
+            this.InnerTestSuiteNodeName = "results/test-suite";
+            this.TestCaseNodeName = "results/test-case";
```

Comment:
these are constants used in a single class. We dont use constants files for such cases. 
WF